app/views/shared/_navbar.html.erb

```diff
@@ -26,7 +26,7 @@
           <%= link_to "Community", chat_link, class: 'nav-link', target: '_blank', rel: 'noreferrer' %>
         </li>
 
-        <li class="divider nav-link">|</li>
+        <li class="divider nav-link">l</li>
```

Comment:
having the letter l here doesn't seem right,  is there a good reason for not using the pipe that was here before?

Reply:
The pipe won't align. I tried my best lol.